CHANGELOG.md

```diff
@@ -570,7 +572,8 @@ section                size        addr
 
 Initial release
 
-[Unreleased]: https://github.com/rust-embedded/cortex-m-rt/compare/v0.6.11...HEAD
+[Unreleased]: https://github.com/rust-embedded/cortex-m-rt/compare/v0.7.0...HEAD
+[v0.7.0]: https://github.com/rust-embedded/cortex-m-rt/compare/v0.6.11...v0.7.0
```

Comment:
Is the lower version correct?

Reply:
Yep, that's when we branched off to the separate 0.6 branch; 0.6.12 and onwards were all backports from master.